packages/app-store/ee/routing-forms/components/SingleForm.tsx

```diff
@@ -362,14 +362,14 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
                     </Button>
                   </div>
                   {!form._count?.responses && (
-                    <Banner
```

Comment:
This was the only instance where the banner component was used, I replaced it with the `<Alert>` component that looks practically similar. However, it doesn't have the dismiss button (I didn't implement it). This dismiss button in the Banner was fake though, it would hide the message, but would be visible the next time. So I think we can do without here 😁